operator-common/src/main/java/io/strimzi/operator/cluster/model/Ca.java

```diff
@@ -386,30 +377,27 @@ public CertAndKey generateSignedCert(String commonName, String organization) thr
                         Base64.getDecoder().decode(secret.getData().get(podName + ".crt")));
             }
 
-            Collection<String> desiredSbjAltNames = subject.subjectAltNames().values();
-            Collection<String> currentSbjAltNames =
-                    getSubjectAltNames(certAndKey.cert());
+            boolean shouldBeRegenerated = false;
```

Comment:
Can't we just the the non-emptiness of `reasons` instead of using this variable?